pantheon/src/main/java/tech/pegasys/pantheon/cli/subcommands/blocks/BlocksSubCommand.java

```diff
@@ -256,4 +322,9 @@ private void outputBlock(final List<Block> blocks) {
     }
     return metricsService;
   }
+
+  @FunctionalInterface
+  public interface ChainImporterFactory {
```

Comment:
Why does this need to be an explicit functional interface? Could we not use a `Supplier`?

Reply:
We could use `Function<PantheonController<T>, ChainImporter<T>>` instead.  But, I prefer making explicit interfaces for reasons of readability and discoverability.  It's a pattern we use pretty extensively in the codebase.

Reply:
Interesting, I find the explicit interface less readable because the parameter looks like it could be a full blown object with more methods. Not a blocking issue though